examples/lusid/instruments/query-instruments.sql

```diff
@@ -0,0 +1,25 @@
+-- ============================================================
+
+-- In this query, we will filter various types of instruments.
+-- These can be uploaded via the other examples in this folder.
+-- We will query three different types of instrument,
+-- and join the results using union.
+
+-- ============================================================
+
+-- Select common columns and filter
+select DisplayName, ClientInternal, DomCcy, LusidInstrumentId
+from Lusid.Instrument.Equity
+where DisplayName = 'Admiral Group'
+
+union
+
+select DisplayName, ClientInternal, DomCcy, LusidInstrumentId
+from Lusid.Instrument.SimpleInstrument
+where ClientInternal = 'CCYGBP'
```

Comment:
Can we try something which is not "CCYGBP"?
CCY_GBP is an internal code for GBP Cash, which is not a SimpleInstrument, so this might be a little confusing.

Reply:
Maybe just remove this cash instrument from the upsert and keep it as just the equities.

Reply:
Data amended to shopping centre data.